### PR TITLE
Added CHW/CW bypass valves

### PR DESCRIPTION
**Pull request by pranay2811**
Added fields:
chilled_water_bypass_valve_command
chilled_water_bypass_valve_status
chilled_water_valve_status
condensing_water_bypass_valve_command
condensing_water_bypass_valve_status

Added Abstracts
CHWBYPVBRC & CWBYPVBRC

### CODE REVIEW

**Comment by pranay2811** [comment]
@shambergoldstein This is ready for your review

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -8113,6 +8125,17 @@ CWBYPVPM:
...
   - MONITORING
 
+CWBYPVBRC:
+  guid: "2cf8239f-7da4-4df7-8125-8f70e5962def"
+  description: "Condensing water bypass valve binary (open/closed) control."
+  is_abstract: true
+  implements: 
+  - OPERATIONAL
+  opt_uses:
+  - condensing_water_bypass_valve_status
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:8135]: please move this to the required uses

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -4159,6 +4160,17 @@ CHWBYPVPM:
...
   - MONITORING
 
+CHWBYPVBRC:
+  guid: "38064bbf-cd53-430d-af16-bf77d7e565a1"
+  description: "Chilled water bypass valve binary (open/closed) control."
+  is_abstract: true
+  implements: 
+  - OPERATIONAL
+  opt_uses:
+  - chilled_water_bypass_valve_status
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4170]: please move this to required uses

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 2 above)
Comment (pranay2811) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (pranay2811) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1]: What does the R represent in the abstract type names? I think it makes sense to change these to CHWBYPVBC and CWBYPVBC (...water bypass valve binary control).

Comment (pranay2811) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1]: I think I saw an abstract type ending in BRC. Looks like the r represents return which is not the case for these abstracts. I will edit them now

Comment (pranay2811) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1]: Done

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**